tests/e2e/tracer/handlers/basic_handler.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+from aws_lambda_powertools import Tracer
+
+tracer = Tracer()
+tracer = Tracer(service="e2e-tests-app")
+
+ANNOTATION_KEY = os.environ["ANNOTATION_KEY"]
+ANNOTATION_VALUE = os.environ["ANNOTATION_VALUE"]
+
```

Comment:
could you also add a test with a synchronous and asynchronous function to be traced?

Reply:
added